src/pages/posthighlight.tsx

```diff
@@ -0,0 +1,134 @@
+import { useEffect, useState } from "react";
```

Comment:
Currently, only the send button is disabled when the summary is being generated. The summarize button should be disabled too.

Reply:
Nevermind. The same state is being used for both the buttons.